### PR TITLE
fixing https://issues.openmrs.org/browse/FM-183

### PR DESCRIPTION
**Pull request by rasanjanap**

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java [hunk 1]
```diff
@@ -24,17 +25,20 @@
...
-	public Condition getByUniqueId(IdDt id) {
-		ConditionService conditionService = Context.getService(ConditionService.class);
-		Condition condition = conditionService.getCondition(id.getIdPart());
-		if (condition == null) {
-			throw new ResourceNotFoundException("Condition is not found for the given Id " + id.getIdPart());
-		}
-		return condition;
-	}
+    public Condition getByUniqueId(IdDt id) {
+        ConditionService conditionService = Context.getService(ConditionService.class);
```
Comment (harsha89) [inline omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java]: have this service defined in moduule context file?

Comment (rasanjanap) [inline omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java]: This is already existing service. Hence no need. I forgot to add it when doing appointment and added now

Comment (harsha89) [inline omod/src/main/java/org/openmrs/module/fhir/resources/FHIRConditionResource.java]: Ah okey :+1:

**Comment by harsha89** [comment]
Please squash commits to one

**Comment by rasanjanap** [comment]
Done!

**Pull request merged**